README.md

```diff
@@ -100,6 +100,14 @@ In addition to the Universal Base Image, we can build images based on:
 
 * Our [system requirements](https://help.sonatype.com/display/NXRM3/System+Requirements) should be taken into account when provisioning the Docker container.
 * Default user is `admin` and the uniquely generated password can be found in the `admin.password` file inside the volume. See [Persistent Data](#user-content-persistent-data) for information about the volume.
+  
+  > You could print the generated admin password by running the following command after the container is finished running:
+  > 
+  > ```sh
+  docker container exec -it $(docker ps -q --filter "name=nexus") cat /nexus-data/admin.password && echo
```

Comment:
Something is funky with the markdown here. When I go and view it on your branch it isn't rendering properly - https://github.com/ahmed-musallam/docker-nexus3/blob/patch-1/README.md#notes

Reply:
ah! my local markdown viewer failed me... fixed!